misc/tcb.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@PREFIX@
+exec_prefix=${prefix}
+libdir=@SLIBDIR@
+includedir=@INCLUDEDIR@
+
+Name: tcb
+Description: Implementation of the tcb password shadowing scheme
```

Comment:
This description doesn't look very descriptive for me.  Yes, libtcb is a part of implementation, but e.g. libnss_tcb is also a part of implementation.

Reply:
Should be fixed by rewording to `Library implementing the tcb password shadowing scheme` in rebased commit.